### PR TITLE
Create an entry in the etl job table if it does not yet exist

### PR DESCRIPTION
**Pull request by hdahme**
@michaelliu2 and @ianlofs 

Creates a new entry in the `_etl_jobs_` table for new etl jobs, if it does not already exist.

### CODE REVIEW

File: etl_framework/BaseEtlSetUp.py [hunk 1]
```diff
@@ -190,7 +196,12 @@ def set_etl_job_cutoff_value(self, datetime_cutoff=None):
         else:
             sql_statement = self.ETL_JOB_SELECT_CUTOFF_STATEMENT%(self.etl_job_id, )
 
-            datetime_cutoff = self.sql_database.run_statement(sql_statement, fetch_data=True)[0][0][0]
+            try:
+                datetime_cutoff = self.sql_database.run_statement(sql_statement, fetch_data=True)[0][0][0]
+            except IndexError:
+                # The ETL job has not yet been created, so we create it here
+                sql_statement = self.ETL_JOB_CREATE_JOB_STATEMENT%(self.etl_job_id, self.etl_job_name, )
+                self.sql_database.run_statement(sql_statement)
```
Comment (ianlofs) [inline etl_framework/BaseEtlSetUp.py]: feels weird using an error to do anything other than error handling

Comment (michaelliu2) [inline etl_framework/BaseEtlSetUp.py]: @ianlofs It's actually pretty pythonic to do exception handling as conditional logic.  I'm worried that `IndexError` might be caused by another issue though. Is there a more specific way of capturing that the row doesn't exist @hdahme ??

Comment (hdahme) [inline etl_framework/BaseEtlSetUp.py]: Updated guys -- thanks

**Comment by ianlofs** [comment]
sweet! thank you 👍

**Comment by michaelliu2** [comment]
@hdahme did you test it just to make sure it works? If so, 👍

**Comment by hdahme** [comment]
Indeed. Thanks guys

**Pull request merged**